server/storage/memory_test.go

```diff
@@ -90,3 +91,135 @@ func TestGetChecksumNotFound(t *testing.T) {
 	require.Error(t, err)
 	require.IsType(t, ErrNotFound{}, err)
 }
+
+func TestMemoryGetChanges(t *testing.T) {
+	s := NewMemStorage()
+
+	// non-int changeID
+	c, err := s.GetChanges("foo", 10, "")
+	require.Error(t, err)
+	require.Len(t, c, 0)
+
+	// add some records
+	require.NoError(t, s.UpdateMany("alpine", []MetaUpdate{
+		{
+			Role:    data.CanonicalTimestampRole,
+			Version: 1,
+			Data:    []byte{},
+		},
+		{
+			Role:    data.CanonicalTimestampRole,
+			Version: 2,
+			Data:    []byte{},
+		},
+		{
+			Role:    data.CanonicalTimestampRole,
+			Version: 3,
+			Data:    []byte{},
+		},
+		{
+			Role:    data.CanonicalTimestampRole,
+			Version: 4,
+			Data:    []byte{},
+		},
+	}))
+	require.NoError(t, s.UpdateMany("busybox", []MetaUpdate{
```

Comment:
Oops sorry, I didn't look far enough - not `tuf_store_test` - `storage_test`.   Looks like we have a delete test in there - could add the check of the change feed in that test instead of in this one.  There is also a rollback test in there - could we add an assertion that in the rollback case, no change log is created for that update?